src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    { !person.isMarried
+      ? (<p className="Person__partner">I am not married</p>)
+      : (<p className="Person__partner">{`${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}</p>)
```

Comment:
dont use nested ternary operator.
In fact, it would be better to move your partner definition into a separate method